bindings/cs/MongoDB.Crypt.Test/BasicTests.cs

```diff
@@ -407,14 +502,15 @@ public void TestLocalKeyCreationStepwise()
 
         static IEnumerable<string> FindTestDirectories()
         {
-            string searchPath = Path.Combine("..", "test", "example");
+            string[] searchPaths = new [] { Path.Combine("..", "test", "example"), Path.Combine("..", "test", "data") };
```

Comment:
The `kms-encrypt-reply.txt` file lives in a different part of the repo, so I had to update this search functions